src/app/container/info-tab/info-tab.component.ts

```diff
@@ -67,7 +65,7 @@ export class InfoTabComponent implements OnInit, OnChanges {
     if (this.selectedVersion && this.tool) {
       this.currentVersion = this.selectedVersion;
       const found = this.validVersions.find((version: Tag) => version.id === this.selectedVersion.id);
-      this.isValidVersion = found ? true : false;
+      this.isValidVersion = !!found;
```

Comment:
You could just use `some` instead of `find` on line 69/67 and assign directly to isValidVersion.